README.md

```diff
@@ -158,6 +158,7 @@ The configuration includes the following parameters:
   * `runtime`: (Optional) runtime of the function, if you need to deploy multiple functions with different runtimes in your Serverless Project. If absent, `provider.runtime` will be used to deploy the function, see [this example project](./examples/multiple).
   * `events` (Optional): List of events to trigger your functions (e.g, trigger a function based on a schedule with `CRONJobs`). See `events` section below
   * `custom_domains` (Optional): List of custom domains, refer to the [how to add a custom domain](https://www.scaleway.com/en/docs/compute/functions/how-to/add-a-custom-domain-name-to-a-function/) documentation page
+  * `httpOption` (Optional): force https redirection, possible values are `enabled` and `redirected` (default: `enabled`)
```

Comment:
Do we have somewhere in Scaleway documentation where the httpOption behaviour is explained ? If sou can you add a link here ?

Reply:
I've checked with Lucas, no there isn't